features/192-asset-gear/index.md

```diff
@@ -506,6 +507,8 @@ To be specified in the future.
 
 The initial implementation does not have caching.
 
+HTTP Range request is not supported because it conflicts with image processing.
```

Comment:
If the only reason is that it conflicts with image processing, we can limit it to when the query string is not present?

For non-trivial sized assets (e.g. videos, large documents), having resumable download may be important.

Reply:
Let me revise it to be

> If image processing query is present, then Accept-Range response from the upstream response is removed, thus disabling range request. If Range is present in the request header, the whole image processing pipeline is skipped.

Reply:
I think this is dangerous:
- `Accept-Range` should be set to `none` instead of being removed.
- The image data is different depending on the presence of `Range` header.

Reply:
> If image processing query is present, then Range is always ignored and Accept-Range is set to none, thus disabling range request